src/Workspaces/Core/Portable/Workspace/Host/EventListener/IWorkspaceEventListenerProvider.cs

```diff
@@ -33,7 +33,7 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
             return new Service(workspace, EventListenerTracker<object>.GetListeners(workspace.Kind, eventListeners));
         }
 
-        private class Service(Workspace workspace, IEnumerable<IEventListener<object>> eventListeners) : IWorkspaceEventListenerService
+        internal class Service(Workspace workspace, IEnumerable<IEventListener<object>> eventListeners) : IWorkspaceEventListenerService
```

Comment:
curious why.

Reply:
For testing. See MockWorkspaceEventListenerProvider

Reply:
wfm.

though our general practice on that is to keep rpviate, and expose a testaccessor.  that way it's clear that normal product code shouldn't call this.